src/main/java/calculator/view/InputView.java

```diff
@@ -0,0 +1,58 @@
+package calculator.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class InputView {
+
+    private static final String CUSTOM_DELIMITER_PREFIX = "//";
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\\\\n(.*)");
+    private static final String DEFAULT_NUMBER_DELIMITER_REGEX = "[,:]";
+
+    private InputView() {
+    }
+
+    public static List<Integer> inputNumbers() {
+        System.out.println("덧셈할 문자열을 입력해 주세요.");
+
+        String input = Console.readLine();
+        if (input.startsWith(CUSTOM_DELIMITER_PREFIX)) {
+            return splitNumbersWithCustomDelimiter(input);
+        }
+        return splitNumbers(input, DEFAULT_NUMBER_DELIMITER_REGEX);
+    }
+
+    private static List<Integer> splitNumbersWithCustomDelimiter(String input) {
+        Matcher matcher = CUSTOM_DELIMITER_PATTERN.matcher(input);
+
+        if (matcher.find()) {
+            String delimiter = matcher.group(1);
+            String numbers = matcher.group(2);
+            return splitNumbers(numbers, Pattern.quote(delimiter));
+        } else {
+            throw new IllegalArgumentException("잘못된 입력 형식입니다.");
+        }
```

Comment:
```suggestion
        if (matcher.find()) {
            String delimiter = matcher.group(1);
            String numbers = matcher.group(2);
            return splitNumbers(numbers, Pattern.quote(delimiter));
        }
        throw new IllegalArgumentException("잘못된 입력 형식입니다.");
```

이렇게 작성하는 것과 코드의 동작 방식이 다를까요??
불필요한 else문을 없애볼 수 있을 것 같아요!~ 😀

Reply:
혹시 위 코드에서 else문을 사용해도 되고 안해도 될 것 같은데, 불필요한 else문을 없애달라고 하신 이유가 있으실까요?

Reply:
클린 코드 책에서도 언급된 내용인데요, 불필요한 else 문을 제거하면 가독성이 좋아져 코드가 더 읽기 쉬워집니다~
또한 보통 else 문이 나오면, else가 어떤 상황에 해당하는지 알아보기 위해 이전 if 조건을 다시 확인해야 하는 단점도 있다 생각해요~

물론 충용님 말씀대로 사용해도 되고 안해도 되지만, 지금부터 else문을 지양하는 연습을 해보시면 좋을 것 같아요~

지금처럼 메서드에서 특정 조건을 만족하면 바로 리턴하도록 작성하는 방식 좋습니다! 👍